src/plugins/FileService/components/UploadDropArea.tsx

```diff
@@ -0,0 +1,131 @@
+        flex: 1,
+        padding: 12,
+        overflow: 'hidden',
+        userSelect: 'none',
+    },
+    here: {
+        fontSize: 14,
+        lineHeight: 2.5,
+        color: '#2CA4EF',
+        userSelect: 'none',
+    },
+    hint: {
+        margin: 0,
+        color: '#939393',
+        fontSize: 12,
+        lineHeight: 1.5,
+        userSelect: 'none',
+    },
+    over: {
+        borderColor: '#2CA4EF',
+        borderStyle: 'solid',
+        userSelect: 'none',
+        '& > $indicator': { opacity: 1 },
+    },
+    indicator: {
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        opacity: 0,
+        cursor: 'pointer',
+        transition: 'all 500ms ease-out',
+        height: '100%',
+        width: '100%',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        background: 'rgba(255, 255, 255, 0.85)',
+        fontSize: 30,
+        color: '#2CA4EF',
+        userSelect: 'none',
+        zIndex: 1,
+    },
+    uploader: {
+        position: 'absolute',
+        top: 38,
+        userSelect: 'none',
+        cursor: 'pointer',
+        zIndex: 0,
+    },
+})
+
+interface Props {
+    maxFileSize: number
+    onFile(file: File): void
+}
+
+export const UploadDropArea: React.FC<Props> = ({ maxFileSize, onFile }) => {
+    const { t } = useI18N()
+    const classes = useStyles()
+    const snackbar = useSnackbar()
+    const [bond, { over }] = useDropArea({
+        onFiles(files) {
+            if (files.length > 1) {
+                onError(101)
+            } else if (files[0].size > maxFileSize) {
+                onError(102)
+            } else {
+                onFile(files[0])
+            }
+        },
+        onText: () => onError(101),
+        onUri: () => onError(101),
+    })
+    const MAX_FILE_SIZE = formatFileSize(maxFileSize)
+    const onInput = (event: React.FormEvent<HTMLInputElement>) => {
+        const file = event.currentTarget.files?.item(0)
+        if (isNil(file)) {
+            onError(101)
+        } else if (file.size > maxFileSize) {
+            onError(102)
+        } else {
+            onFile(file)
+        }
+    }
+    const onError = (code: number) => {
+        const messages: Record<number, string> = {
+            101: t('plugin_file_service_error_101'),
+            102: t('plugin_file_service_error_102', { limit: MAX_FILE_SIZE }),
+        }
+        if (code in messages) {
+            snackbar.enqueueSnackbar(`Error ${code}: ${messages[code]}`, { variant: 'error' })
+        }
+    }
+    return (
+        <Typography component="label" {...bond} className={classNames(classes.label, { [classes.over]: over })}>
+            <input type="file" onInput={onInput} hidden />
+            <section className={classes.indicator}>{t('plugin_file_service_drop_indicator')}</section>
+            <Image
+                className={classes.uploader}
+                src={getUrl('/plugin/file-service/indicator.svg')}
```

Comment:
Please don't use this utility to render SVG. You can change the SVG into a TSX file and get a high-resolution icon. And if possible, please use the icon from [@material-ui/icons](https://material-ui.com/components/material-icons/) or [react-feather](https://feathericons.com/).